Objects/bytearrayobject.c

```diff
@@ -1388,6 +1393,31 @@ bytearray_removesuffix_impl(PyByteArrayObject *self, Py_buffer *suffix)
 }
 
 
+/*[clinic input]
+bytearray.resize
+    size: Py_ssize_t
+        New size to resize to..
+    /
+Resize the internal buffer of bytearray to len.
+[clinic start generated code]*/
+
+static PyObject *
+bytearray_resize_impl(PyByteArrayObject *self, Py_ssize_t size)
+/*[clinic end generated code: output=f73524922990b2d9 input=75fd4d17c4aa47d3]*/
+{
+    Py_ssize_t start_size = PyByteArray_GET_SIZE(self);
+    int result = PyByteArray_Resize((PyObject *)self, size);
+    if (result == -1) {
```

Comment:
The convention for us is to use `< 0`
```suggestion
    if (result < 0) {
```